src/app/src/BackToMapButton.js

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react';
+
+import { initialViewportState } from './map.reducers';
+import './BackToMapButton.scss';
+
+export default class BackToMapButton extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.handleOnClick = this.handleOnClick.bind(this);
+    }
+
+    handleOnClick() {
+        this.props.handleOnClick(initialViewportState);
+    }
```

Comment:
Nitpick but I think this could be a function component, and the `handleOnClick` method could just be set up like:

```
const handleClick = () => this.props.handleOnClick(initialViewportState);
```

Reply:
Nice!